### DISCUSSION TITLE
Why there is no steppers in vuetify 3?

### DISCUSSION THREAD

**Discussion by gaolei8888** [vuetifyjs/vuetify#16676] [Q&A]
Why there is no steppers in vuetify 3? Is there a plan to release it in future releases?

**Comment by 1Luc1** [comment]
v-stepper seems to be within milestone [v3.3.0](https://github.com/vuetifyjs/vuetify/milestone/61)
the corresponding task for v-steppers is here: https://github.com/vuetifyjs/vuetify/issues/13509
